packages/dbml-core/src/parse/mysql/parser.pegjs

```diff
@@ -491,7 +491,7 @@ factor = factors:(character+ _ "(" expression ")"
     / (exprCharNoCommaSpace+ &(_/","/");"/endline");")) / exprChar+ &.) {
     	return _.flattenDeep(factors).join("");
     }   
-exprChar = [\',.a-z0-9_+-\`]i
+exprChar = [\',.a-z0-9_+-\`%]i
```

Comment:
How about other characters `^?&*...` anh?

Reply:
I think we just need to cover `%` in this case because `%` is usually used as unit of measurement.
I haven't seen use cases for other characters so until the case arises, we can support them later.

Reply:
ok anh